tests/test_system_swap.py

```diff
@@ -0,0 +1,32 @@
+import mock
+import psutil
+
+from tests.common import AgentCheckTest
+
+B2MB  = float(1048576)
+
+class _PSUtilSwapStatsMock(object):
+    def __init__(self, sin, sout):
+        self.sin = sin
+        self.sout = sout
+
+ORIG_SWAP_IN = 115332743168
+ORIG_SWAP_OUT = 22920884224
+
+SWAP_IN_INCR = 2
+SWAP_OUT_INCR = 4
+
+MOCK_PSUTIL_SWAP_STATS = [
+    _PSUtilSwapStatsMock(ORIG_SWAP_IN, ORIG_SWAP_OUT),
+    _PSUtilSwapStatsMock(ORIG_SWAP_IN + SWAP_IN_INCR, ORIG_SWAP_OUT + SWAP_OUT_INCR),
+]
+
+class SystemSwapTestCase(AgentCheckTest):
+
+    CHECK_NAME = 'system_swap'
+
+    @mock.patch('psutil.swap_memory', side_effect=MOCK_PSUTIL_SWAP_STATS)
+    def test_system_swap(self, mock_swap_stats):
+        self.run_check_twice({"instances": [{}]})
+        self.assertMetric('system.swap.swapped_in', value=SWAP_IN_INCR, count=1)
```

Comment:
@remh it looks like `run_check_twice` explicitly waits a second between checks which is why the rate evaluates correctly when I run the test locally
```
    def run_check_twice(self, config, agent_config=None, mocks=None,
                        force_reload=False):
        self.run_check(config, agent_config, mocks, force_reload)
        time.sleep(1)
        self.run_check(config, agent_config, mocks)
```